### PR TITLE
DTSSTCI-981: Added integration tests for Caseworker edit record listing and panel composition

### PR DESCRIPTION
**Pull request by rajaram-nagarajan**
### Change description ###

Added integration tests

### JIRA link ###

https://tools.hmcts.net/jira/browse/DTSSTCI-981

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)
- [ ] `enable-e2e-tests` label can be used to run the e2e tests before QA handover and before release (required)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket has been reviewed by QA
- [ ] the user story has been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerEditRecordListingIT.java [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+            .getResponse()
+            .getContentAsString();
+
+        assertThatJson(response)
+            .inPath(CONFIRMATION_HEADER)
+            .isString()
+            .contains("""
+                # Listing record updated\s
+                ##  If any changes are made to this hearing,  remember to make those changes in this listing record.\s
+                ## A notification has been sent to""");
```
Comment (tomxelliott) [inline src/integrationTest/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerEditRecordListingIT.java:260]: Are we able to test sending to all parties? There is an example in one of the integration tests I have merged.

Comment (rajaram-nagarajan) [reply src/integrationTest/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerEditRecordListingIT.java]: Thanks, it's done.

File: src/integrationTest/resources/responses/caseworker-edit-record-listing-about-to-submit-response.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      }
+    ],
+    "closedDayCount": "",
+    "currentEvent": "",
+    "firstHearingDate": "14 Aug 2024",
+    "dataVersion": 0,
+    "cicCaseSelectedDocument": {}
+  },
+  "errors": [
+    "One party must be selected."
```
Comment (gmmagruder) [inline src/integrationTest/resources/responses/caseworker-edit-record-listing-about-to-submit-response.json:70]: Should an error be returned for this test? It seems like you are testing happy path here.

Comment (rajaram-nagarajan) [reply src/integrationTest/resources/responses/caseworker-edit-record-listing-about-to-submit-response.json:70]: This is just a generic response. This is not the output from the test case. I can remove the errors section if it is confusing.

**Review by tomxelliott — APPROVED** [review]

**Review by gmmagruder — APPROVED** [review]

**Pull request merged**